src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -962,7 +958,28 @@ function getWarningMessage() {
  * @param textAreaId - Id of text area for which char are to be counted
  * @param charCountId - Id of Label to display length of text area
  */
-function updateTextQuestionCharCount(textAreaId, charCountId) {
-    var cs = $('#' + textAreaId).val().length;
-    $('#' + charCountId).text(cs);
+function updateTextQuestionWordsCount(textAreaId, wordsCountId, recommendedLength) {
+	
+    var response = $('#' + textAreaId).val();
+    var $wordsCountElement = $('#' + wordsCountId);
+    var wordsCountElementText = $('#' + wordsCountId).text();
+
+    var wordsCount = response.split(/\s/g).filter(function(item) {
+        return item.match(/\w/);
+    }).length;
+
+    if (response.length === 0) {
+        $wordsCountElement.text('0');
+    } else {
+        $wordsCountElement.text(wordsCount);
+    }
+
+    var upperLimit = recommendedLength + recommendedLength * 0.1;
+    var lowerLimit = recommendedLength - recommendedLength * 0.1;
+
+    if (wordsCountElementText > lowerLimit && wordsCountElementText < upperLimit) {
```

Comment:
I realised we could just use `wordsCount` here too, instead of `wordsCountElementText` :disappointed: